mask_cyclegan_vc/model.py

```diff
@@ -327,11 +327,11 @@ def __init__(self, input_shape=(80, 64), residual_in_channels=256):
                                                        padding=[0, 1]))
 
     def downsample(self, in_channels, out_channels, kernel_size, stride, padding):
-        convLayer = nn.Sequential(nn.Conv2d(in_channels=in_channels,
+        convLayer = nn.Sequential(nn.utils.spectral_norm(nn.Conv2d(in_channels=in_channels,
```

Comment:
Please remove spectral norm for now. Perhaps it can be added as an arg?

Reply:
This modification not resulted in perceptible improvement, therefore I remove it.